.alexrc

```diff
@@ -0,0 +1,3 @@
+{
+  "allow": ["execute", "host-hostess", "remains", "reject", "traps", "invalid", "period", "invalid", "execution", "simple", "failure"]
```

Comment:
invalid is twice in this list, please check.

Reply:
Addressed. Done.